<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>linkedList_admin</class>
 <widget class="QMainWindow" name="linkedList_admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>788</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>611</width>
      <height>191</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>609</width>
       <height>189</height>
      </rect>
     </property>
     <widget class="QTreeWidget" name="Tittle">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>191</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(219, 99, 44);</string>
      </property>
      <column>
       <property name="text">
        <string>                           Tittle</string>
       </property>
      </column>
     </widget>
     <widget class="QTreeWidget" name="Year">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>0</y>
        <width>201</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(219, 99, 44);</string>
      </property>
      <property name="lineWidth">
       <number>1</number>
      </property>
      <column>
       <property name="text">
        <string>                              Year</string>
       </property>
      </column>
     </widget>
     <widget class="QTreeWidget" name="Author">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>0</y>
        <width>201</width>
        <height>191</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(219, 99, 44);</string>
      </property>
      <column>
       <property name="text">
        <string>                           Author</string>
       </property>
      </column>
     </widget>
     <widget class="QScrollBar" name="verticalScrollBar">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>0</y>
        <width>16</width>
        <height>191</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_books_Tittle">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>171</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(219, 99, 44);a</string>
      </property>
      <property name="flow">
       <enum>QListView::TopToBottom</enum>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignJustify</set>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_books_Author">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>30</y>
        <width>171</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(219, 99, 44);</string>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignJustify</set>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget_books_year">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>30</y>
        <width>171</width>
        <height>151</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>15</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">selection-background-color: rgb(219, 99, 44);</string>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignJustify</set>
      </property>
     </widget>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>0</y>
        <width>20</width>
        <height>191</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>0</y>
        <width>20</width>
        <height>191</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>150</y>
      <width>252</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButton_add_book_admin">
       <property name="text">
        <string>Tambahkan</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_delete_book_admin">
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton_look_book_admin">
       <property name="text">
        <string>Lihat</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-6</x>
      <y>-6</y>
      <width>801</width>
      <height>501</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="Resource.qrc">:/Resource/Image/ONLINE PLAYBOOK (2).png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>410</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
   <zorder>label</zorder>
   <zorder>layoutWidget</zorder>
   <zorder>scrollArea</zorder>
   <zorder>pushButton</zorder>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
